.github/workflows/k3d-nightly-ci.yaml

```diff
@@ -1,12 +1,18 @@
 name: Nightly tests (K3d)
 
 on:
+  push:
+    branches:
+      - issue-2673-seed-file-backend-support
```

Comment:
```suggestion
```

To remove before merging - added to test seed file cleanup jobs in PR

Reply:
Can remove this now - can still trigger nightly run on demand via GitHub